src/Umbraco.Web.UI.Client/src/common/services/tinymce.service.js

```diff
@@ -333,10 +301,17 @@ function tinyMceService($rootScope, $q, imageHelper, $locale, $http, $timeout, s
 
                     images_upload_handler: uploadImageHandler,
                     automatic_uploads: false,
-                    images_replace_blob_uris: false,
-                    init_instance_callback: initEvents
+                    images_replace_blob_uris: false
                 };
-
+                
+                if (args.htmlId) {
+                    config.selector = "#" + args.htmlId;
```

Comment:
Do we still use/support the HTML ID, I thought you had removed it in this PR?!

Reply:
I dont know about third party plugins, so keeping it backwards compatible.